keyboards/skyloong/gk87/q1/led_hc595.c

```diff
@@ -0,0 +1,80 @@
+//  Copyright 2023 NaturalZh (@NaturalZh)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+#ifndef HC595_ST_PIN
+#    error hc595: no storage register clock pins defined!
+#endif
+
+#ifndef HC595_SH_PIN
+#    error hc595: no shift register clock pins defined!
+#endif
+
+#ifndef HC595_DS
+#    error hc595: no serial data input pins defined!
+#endif
+
+/*
+ *LED display driver.74HC595 Serial to parallel control LED field on/off.
+ *  data(default):1-on/0-off (left low bit)
+ *    bit0 --- NUM indicator
+ *    bit1 --- CAPS indicator
+ *    bit2 --- SCR indicator
+ *    bit3 --- WinLock indicator
+ *    bit4 --- Skyloong LOGO display
+ *    bit5 --- Win layer indicator
+ *    bit6 --- MAC layer indicator
+ */
+#define NUM_ON 0b10000000
+#define CAPS_ON 0b01000000
+#define SCR_ON 0b00100000
+#define WINLK_ON 0b00010000
+#define SKYLOONG 0b00001000
+#define WIN_ON 0b00000010
+#define MAC_ON 0b00000100
+
+#define CLOCK_TIME 15
+
+static inline void setPinOutput_writeLow(pin_t pin) {
```

Comment:
Mostly a nitpick, but these should be renamed to match the gpio function names (eg `gpio_set_pin_output_write_low`  or similar.  Eg, they should be consistent.

Reply:
OK,I see and update it.
Please check it again,thanks. @drashna